Sources/NIOSSH/Role.swift

```diff
@@ -14,6 +14,10 @@
 
 /// The role of a given party in an SSH connection.
 public enum SSHConnectionRole {
+    public static let bundledTransportProtectionSchemes: [NIOSSHTransportProtection.Type] = [
```

Comment:
I don't think this makes sense on `SSHConnectionRole`. This should go on config objects. We can avoid some duplication by using an internal central location.

Reply:
moved to `Constants`